spec/api/v2/schemas/metadata.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    module Schemas
+      module Metadata
+        UUID = { type: :string, format: :uuid }.freeze
+        METADATA = {
+          type: 'object',
+          properties: {
+            total: {
+              type: 'number',
+              examples: [1]
+            },
+            limit: {
+              type: 'number',
+              maximum: 100,
+              minimum: 1,
+              default: 10,
+              examples: [10, 100]
+            },
+            offset: {
+              type: 'number',
+              minimum: 1,
+              default: 1,
+              examples: [1, 3]
+            },
+            sort_by: {
+              type: 'string',
+              examples: %w[asc desc],
+              default: 'version:asc'
+            },
+            # FIXME: compatibility with the V1 logic from the common concerns
+            relationships: {
+              type: 'boolean',
+              examples: [false, true]
+            },
```

Comment:
Nope, you can drop this

Reply:
The new endpoint still generates `relationships`.

Reply:
How and why? The parameter handling doesn't accept this...

Reply:
^^ constant